bigtable/tests/system.py

```diff
@@ -84,7 +84,7 @@ def setUpModule():
         credentials = EmulatorCreds()
         Config.CLIENT = Client(admin=True, credentials=credentials)
     else:
-        Config.CLIENT = Client(admin=True)
+        Config.CLIENT = Client(project='grass-clump-479', admin=True)
```

Comment:
You can't set this project here.  There's probably a different way to use the project you care about.

Reply:
@sduskis I forgot to remove that. I fixed it.